bobtemplates/plone/view/views/+view_name_normalized+.pt.example.bob

```diff
@@ -0,0 +1,12 @@
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:metal="http://xml.zope.org/namespaces/metal"
+      xmlns:tal="http://xml.zope.org/namespaces/tal"
+      xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+      metal:use-macro="context/main_template/macros/master">
+<body>
+    <metal:block fill-slot="content-core">
+        <h1>Sample View</h1>
```

Comment:
add another tag here which actually calls your view method.
Also use i18n-translate here as example.
Remember this code should be also a good basic example of how to write a view.